contrib/checkstyle.xml

```diff
@@ -141,9 +141,11 @@
     <module name="OverloadMethodsDeclarationOrder"/>
     <module name="VariableDeclarationUsageDistance"/>
     <module name="CustomImportOrder">
-      <property name="severity" value="ignore"/>
-      <property name="customImportOrderRules" value="STATIC###SPECIAL_IMPORTS###THIRD_PARTY_PACKAGE###STANDARD_JAVA_PACKAGE"/>
-      <property name="specialImportsRegExp" value="com.google"/>
+      <property name="severity" value="error"/>
```

Comment:
Should we align all projects to have similar properties?? i.e.:
```
      <property name="option" value="above"/>
      <property name="groups" value="emissary,*,java"/>
      <property name="ordered" value="true"/>
      <property name="separated" value="true"/>
      <property name="separatedStaticGroups" value="false"/>
      <property name="sortStaticImportsAlphabetically" value="true"/>
```

Reply:
When you say "all projects" I think you mean just this one and one other project, but I'm not sure that there might be more projects.  What inconsistency are you seeing?  Between the two pull requests for this issue, there should be consistency.